frigate/config.py

```diff
@@ -961,8 +961,16 @@ def runtime_config(self, plus_api: PlusApi = None) -> FrigateConfig:
                     if "detect" in input.roles:
                         try:
                             stream_info = get_video_properties(input.path)
-                            camera_config.detect.width = stream_info["width"]
-                            camera_config.detect.height = stream_info["height"]
+                            camera_config.detect.width = (
```

Comment:
let's simplify this code and move this logic outside of the try / except and have the except clause set stream_info to {"width": 0, "height": 0}

that way the values are only set to the config in one place

Reply:
Do you mean that I should manually generate an exception in get_video_properties?

Reply:
No, this is what I am suggesting

```python
                        try:
                            stream_info = get_video_properties(input.path)
                        except Exception:
                            stream_info = {"width": 0, "height": 0}

                        if stream_info["width"] == 0 or stream_info["height"] == 0:
                            logger.warn(
                                f"Error detecting stream resolution automatically for {input.path} Applying default values."
                            )

                        camera_config.detect.width = (
                           stream_info["width"]
                            if stream_info["width"] > 0
                            else DEFAULT_DETECT_DIMENSIONS["width"]
                        )
                        camera_config.detect.height = (
                            stream_info["height"]
                            if stream_info["height"] > 0
                            else DEFAULT_DETECT_DIMENSIONS["height"]
                        )
```